src/ExtendedBehavior.vala

```diff
@@ -72,6 +72,23 @@ public interface PantheonWayland.ExtendedBehavior : Gtk.Widget, Gtk.Native {
         }
     }
 
+    /**
+     * Tells the wm to make this surface a monitor label. This will cause the surface to be
+     * positioned in the top left corner of the given monitor. The surface will also be kept
+     * above other surfaces.
+     * Should only be called after {@link connect_to_shell}.
+     *
+     * @param monitor_index the index of the monitor this surface will label
+     */
+    public void make_monitor_label (int monitor_index) {
```

Comment:
I guess we should move this to the next to `focus()` so that the order matches with xml and vapi?